BPA/Ch.07/cjs_mnl_ran.stan

```diff
@@ -24,47 +24,50 @@ transformed parameters {
   vector<lower=0,upper=1>[n_occasions - 1] q;
   simplex[n_occasions] pr[n_occasions - 1];
 
-  mu <- logit(mean_phi);
+  mu = logit(mean_phi);
 
   // Constraints
   for (t in 1:(n_occasions - 1)) {
-    phi[t] <- inv_logit(mu + epsilon[t]);
-    p[t] <- mean_p;
+    phi[t] = inv_logit(mu + epsilon[t]);
```

Comment:
We have a lot more functions vectorized, so if you can't work on the logit scale and just define

```
logit_phi = mu + epsilon;
```

then you can do this:

```
phi = inv_logit(mu + epsilon);
```

though you may need to slice down to `1:(n_occassions - 1)` if that isn't the size of `mu`.